Arithmetic_Progression/README.md

```diff
@@ -0,0 +1,59 @@
+# Arithmetic Progression (A.P)
```

Comment:
Do add a single line after all the headings. I never told you to remove those.

Reply:
Ohh. I mistaken it.